platform/ui/src/views/StudyList/components/StudyListTable.js

```diff
@@ -170,27 +185,85 @@ const TableRow = () => {
   );
 };
 
-const StudyListTable = () => {
+TableRow.propTypes = {
+  AccessionNumber: PropTypes.string.isRequired,
+  Modalities: PropTypes.string.isRequired,
+  Instances: PropTypes.number.isRequired,
+  PatientId: PropTypes.string.isRequired,
+  PatientName: PropTypes.string.isRequired,
+  StudyDescription: PropTypes.string.isRequired,
+  StudyDate: PropTypes.string.isRequired,
+  series: PropTypes.array.isRequired,
+};
+
+const StudyListTable = ({ studies, numOfStudies }) => {
+  const renderTable = () => {
+    return (
+      <table className="w-full text-white">
+        <tbody>
+          {studies.map((study, i) => (
+            <TableRow
+              key={i}
+              AccessionNumber={study.AccessionNumber || ''}
+              Modalities={study.Modalities || ''}
+              Instances={study.Instances || ''}
+              StudyDescription={study.StudyDescription || ''}
+              PatientId={study.PatientId || ''}
+              PatientName={study.PatientName || ''}
+              StudyDate={study.StudyDate || ''}
+              series={study.series || []}
+            />
+          ))}
+        </tbody>
+      </table>
+    );
+  };
+
+  const renderEmpty = () => {
+    return (
+      <div className="flex flex-col items-center justify-center pt-48">
+        <Icon name="magnifier" className="mb-4" />
+        <Typography className="text-custom-aquaBright" variant="h5">
+          No studies available
+        </Typography>
+      </div>
+    );
+  };
+
   return (
     <>
       <div className="bg-black">
         <div className="container m-auto relative">
-          <div className="bg-custom-blue text-center text-base py-1 rounded-b sticky top-0">
-            <p className="text-white">
-              Filter list to 100 studies or less to enable sorting
-            </p>
-          </div>
-          <table className="w-full text-white">
-            <tbody>
-              {new Array(30).fill('').map((empty, i) => (
-                <TableRow key={i} />
-              ))}
-            </tbody>
-          </table>
+          {numOfStudies > 100 && (
+            <div className="bg-custom-blue text-center text-base py-1 rounded-b sticky top-0">
+              <p className="text-white">
+                Filter list to 100 studies or less to enable sorting
+              </p>
+            </div>
+          )}
+
+          {numOfStudies > 0 && renderTable()}
+          {numOfStudies === 0 && renderEmpty()}
         </div>
       </div>
     </>
   );
 };
 
+StudyListTable.propTypes = {
+  studies: PropTypes.arrayOf(
+    PropTypes.shape({
+      AccessionNumber: PropTypes.string.isRequired,
+      Modalities: PropTypes.string.isRequired,
+      Instances: PropTypes.number.isRequired,
+      PatientId: PropTypes.string.isRequired,
+      PatientName: PropTypes.string.isRequired,
+      StudyDescription: PropTypes.string.isRequired,
+      StudyDate: PropTypes.string.isRequired,
+      series: PropTypes.array.isRequired,
```

Comment:
Need to discuss DTO naming convention with @JamesAPetts

Reply:
Gist is, we enforce capitalization of DICOM Tag names to be consistent with DICOM naming conventions; but does that change when the value deviates from what is stored?

IE. These are "display" representations of the original values. This object is a "data transfer object" (DTO), or you can think of it as a "view model"

Reply:
IMO to avoid confusing we shouldn't just change the casing of these then. Anything non-naturalized DICOM tag format should be explictly stated? I.e. `studyDateDisplay` ? Don't care too much on the naming convention but anything which is formatted to be readable etc needs to be firmly differentiated?

Reply:
That makes sense. So consider tag names "reserved" unless they're immutable value representations?

Reply:
I don't think they need to be immutable, there are scenarios where you want to "fix" the headers.